datumaro/plugins/imagenet_txt_format.py

```diff
@@ -87,17 +102,34 @@ def build_cmdline_parser(cls, **kwargs):
         parser = super().build_cmdline_parser(**kwargs)
         parser.add_argument('--labels-file', dest='labels',
             help="Path to the file with label descriptions (synsets.txt)")
+        parser.add_argument('--no-labels-file', dest='labels',
+            action='store_const', const=(),
+            help="Do not load a file with label descriptions")
+        parser.add_argument('--auto-labels', action='store_true',
+            help="Automatically create generic label descriptions "
+                "if an annotation refers to a label "
+                "for which no description exists")
         return parser
 
     @classmethod
     def find_sources_with_params(cls, path, **extra_params):
```

Comment:
How about passing extra params as named args? This will automatically document the code and add checks for unsupported agruments.